bundles/org.openhab.binding.dirigera/src/main/resources/OH-INF/thing/blind.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dirigera"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="blind">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+
+		<label>Blind</label>
+		<description>Window or door blind</description>
+
+		<channels>
+			<channel id="blind-state" typeId="blind-state">
```

Comment:
Is this `state` id future proof? Currently it only seems to accept up/down/stopped, but would it also be usable for top-down / bottom-up or other variants? Not sure if this makes sense.

Reply:
That's why it's designed as a number with mappings which can be extended.

Also here [these values](https://github.com/dvdgeisler/DirigeraClient/blob/a760b4419a8b1adf469d14a6ce4e750e52d4d540/dirigera-client-api/src/main/java/de/dvdgeisler/iot/dirigera/client/api/model/device/blinds/BlindsState.java#L4) are not coming out of the blue.